examples/transfers/create_crypto_address.py

```diff
@@ -0,0 +1,42 @@
+# Copyright 2025-present Coinbase Global, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+import argparse
+from intx_sdk import IntxServicesClient
+from intx_sdk.services.transfers import CreateCryptoAddressRequest
+
+
+def main():
+    parser = argparse.ArgumentParser(description="Create crypto address")
+    parser.add_argument("--portfolio", required=True, help="Portfolio ID")
+    parser.add_argument("--asset", required=True, help="Asset symbol")
+    parser.add_argument("--network-arn-id", required=True, help="Network ARN ID")
```

Comment:
add example since this is complicated 
Example:
"networks/ethereum-mainnet/assets/313ef8a9-ae5a-5f2f-8a56-572c0e2a4d5a"